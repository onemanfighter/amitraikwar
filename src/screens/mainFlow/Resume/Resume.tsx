import { Box, Text } from '@chakra-ui/react';

const Resume = () => {
  return (
    <Box>
      <Text>Resume</Text>
    </Box>
  );
};

export default Resume;
