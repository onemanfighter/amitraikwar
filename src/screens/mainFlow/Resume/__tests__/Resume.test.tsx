import { render } from '@testing-library/react';
import Resume from '../Resume';

describe('Resume', () => {
  it('should render correctly', () => {
    const { container } = render(<Resume />);

    expect(container).toMatchSnapshot();
  });
});
