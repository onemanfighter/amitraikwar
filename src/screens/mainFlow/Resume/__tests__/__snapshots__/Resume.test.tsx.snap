// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume should render correctly 1`] = `
<div>
  <div
    class="css-0"
  >
    <p
      class="chakra-text css-0"
    >
      Resume
    </p>
  </div>
</div>
`;
